### PR TITLE
fix TFLOPs calculation

### PR DESCRIPTION
**Pull request by polisettyvarma**
when GQA used, we observe right TFLOPs after this fix.
huge difference in TFLOPs is solved for selective recompute when GQA is not used.
some other minor difference will also be observed as logits macs also added.

### CODE REVIEW

**Comment by polisettyvarma** [comment]
same PR in another PR - https://github.com/microsoft/Megatron-DeepSpeed/pull/371
